dowhy/causal_prediction/dataloaders/fast_data_loader.py

```diff
@@ -0,0 +1,68 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
```

Comment:
need a similar header to say that functions are taken from domainbed

Reply:
This file is taken as is from domainbed, similar to misc. Still need to add the header, or the copyright suffices?

Reply:
Have retained the copyright for files where functions/classes are taken as is from domainbed e.g., networks.py, misc.py. Is it required to add a header too for them?